README.md

```diff
@@ -0,0 +1,425 @@
+                selinux: True
+                osg_java: True
+                packages:
+                  - osg-gridftp
+                  - edg-mkgridmap
+                  - rsv
+              - label: BeStMan
+                selinux: False
+                osg_java: True
+                packages:
+                  - osg-se-bestman
+                  - rsv
+              - label: VOMS
+                selinux: False
+                osg_java: True
+                packages:
+                  - osg-voms
+                  - rsv
+              - label: GUMS
+                selinux: False
+                osg_java: True
+                packages:
+                  - osg-gums
+                  - rsv
+6.  Submit the test run (you must submit the DAG from the run directory):
+
+        [user@client ~]$ ./master-run.dag
+7.  Monitor the test run (as desired):
+
+        [user@client ~]$ condor_q -dag
+8.  When the test run finishes, an email will go out to members of the software team (hardcoded in `email-analysis`). In the e-mail will be links to the web interface which will often not work immediately because the test results only get transferred over every 15 minutes.
+
+Troubleshooting
+---------------
+
+### Missing Unicode Fonts
+
+The HTML reports for the testing results utilize unicode characters (namely the padlock to represent that SELinux was enabled). If these characters are appearing as the character code in a block, that means that the font you're using does not support these characters. To render these characters properly, perform the following steps:
+
+1.  Download a suitable Unicode Emoji font. We have had success with the "Noto Emoji" font available from <https://www.google.com/get/noto/>
+2.  Create a `~/.fonts/` dir if one does not already exist
+3.  Copy the `.ttf` file from the downloaded font `.zip` file into `~/.fonts/`
+4.  Run `fc-cache -f`
+
+Notes about running osg-test as VM jobs
+---------------------------------------
+
+This page is my (cat) collected notes and ideas about running osg-test runs as VM jobs, either locally (e.g., CHTC) or in OSG.
+
+### Using VM Universe in HTCondor
+
+For HTCondor 8.4, the documentation for VM universe is in [section 2.11](http://research.cs.wisc.edu/htcondor/manual/v8.4/2_11Virtual_Machine.html) of [the manual](http://research.cs.wisc.edu/htcondor/manual/v8.4/ref.html).
+
+Some quick notes:
+
+-   `universe = vm`
+-   `executable` is just a label
+-   Omit `input`, `output`, and `error`, as they are not used and will cause submit failures
+-   Must select a `vm_type` of `vmware`, `xen`, or `kvm`
+
+### Using VM Universe in CHTC
+
+The only VM universe support in CHTC was using a now-archaic version of [http://www.vmware.com VMware](http://www.vmware.com VMware); this was set up in support of the Thomas Jahns lab. Essentially, in July 2013, there is no current support. However, the CHTC infrastructure team is interested in adding real support for at least kvm.
```

Comment:
The links here did not get translated properly. Should be `[VMware](http://www.vmware.com)`